app/views/jobs/_form.html.erb

```diff
@@ -0,0 +1,89 @@
+<%= fields_for :job, job do |f|  %>
+  <%= hidden_field_tag "job[generic_type]", @job.generic_type %>
+
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Stellenanzeige</label>
+        <%= f.text_field :title, class: "form-control" %>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Externe ID</label>
+        <%= f.text_field :external_id, class: "form-control" %>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Art der Anstellung</label>
+        <%= text_field_tag "job_payload_employment_type", (@job.payload || {})["employement_type"], class: "form-control", name: "job[payload][employment_type]" %>
+      </div>
+    </div>
+  </div>
+
+  <%= render partial: "shared/partials/contacts_form", locals: { record: job, form: f, fields: [] } %>
+
+  <div class="row">
+    <div class="col">
+      <h2 class="d-sm-flex align-items-center justify-content-between my-4">
+        Inhalt
+      </h2>
+
+      <%= render partial: "shared/partials/content_blocks_form", locals: { record: job, form: f, fields: ['body']  } %>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Veröffentlichungsdatum</label>
+        <%=
+          f.date_field :publication_date,
+          class: "form-control",
+          value: f.object.publication_date.try(:to_date) || Time.now.to_date
+        %>
```

Comment:
würde man in `erb` eher wie folgt umbrechen:

```
<%= f.date_field(
      :publication_date,
      class: "form-control",
      value: f.object.publication_date.try(:to_date) || Time.now.to_date
    ) %>
```

Reply:
Ist angepasst